shared/dev/dumb-sheet.render.native.js

```diff
@@ -3,25 +3,37 @@ import React, {Component} from 'react'
 import debounce from 'lodash/debounce'
 import dumbComponentMap from './dumb-component-map.native'
 import type {Props} from './dumb-sheet.render'
-import {Box, Text, Input, Button, NativeScrollView} from '../common-adapters/index.native'
+import {Box, Text, SmallInput, Button, NativeScrollView} from '../common-adapters/index.native'
 import {globalStyles} from '../styles/style-guide'
 
 class Render extends Component<void, Props, any> {
   state: any;
   _onFilterChange: (a: any) => void;
+  _onFilterChangeProp: (a: any) => void;
```

Comment:
i fixed this thing up a little while working on it. we debounce the filter we apply to the view but we want an up to date version of our input